### PR TITLE
[keyboard] bastardkb: restructure folder hierarchy

### PR DESCRIPTION
**Pull request by 0xcharly**
Upstream changes to Bastard Keyboard boards.

This PR:

- Restructure folder hierarchy ahead of supporting other adapters/mcus
- Add support for 2 new keyboards:
  - Charybdis Mini (aka. Cmini, aka. Charybdis 3x6)
  - Dilemma
- Add support for adapter v2
- Add support for RP2040-based Splinky MCU

This PR is on behalf of @bstiq.

## Description

Bastard Keyboards support for the following (adapter, mcu) pairs:

- [`v2/elitec`](https://github.com/Bastardkb/bastardkb-qmk/commit/40c5d056a3b3fac64d57ded3282701de8cf19509)
- [`v2/stemcell`](https://github.com/Bastardkb/bastardkb-qmk/tree/bkb-stemcell)
- [`v2/splinky`](https://github.com/Bastardkb/bastardkb-qmk/tree/bkb-rp2040)
- [`blackpill`](https://github.com/Bastardkb/bastardkb-qmk/commit/965b881baee82b587e17b91bcdf4e863d4ed9fc5)

This PR contains the following changes:

- Move previous implementation to a sub `v1/elitec` folder
- Add support for adapter v2
- Add support for Charybdis 3x6 format
- Add support for Dilemma
- Add support for RP2040-based MCU across all Bastard Keyboards
- Move keyboard USB IDs and strings to data driven
- Update headers to keep maintainers list up to date
- Run `qmk format-c`
- Minor no-op cleanups

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by 0xcharly** [comment]
A couple updates:

- Split the PR into 2 commits to make it easier to review: moved the changes from `qmk format-c` into a separate commit
- Rebased onto latest `qmk:master`

File: keyboards/bastardkb/charybdis/3x5/config.h [hunk 1]
```diff
@@ -18,55 +18,25 @@
...
-#define BOOTMAGIC_LITE_ROW 0
-#define BOOTMAGIC_LITE_COLUMN 0
-#define BOOTMAGIC_LITE_ROW_RIGHT 4
-#define BOOTMAGIC_LITE_COLUMN_RIGHT 0
-
-/* serial.c configuration (for split keyboard) */
-#define SOFT_SERIAL_PIN D2
+/* Disable action features. */
+#define NO_ACTION_MACRO    // Disable old-style macro handling.
+#define NO_ACTION_FUNCTION // Disable old-style function handling.
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x5/config.h:31]: These no longer exist, and can be completely removed. 
```suggestion
```

File: keyboards/bastardkb/charybdis/4x6/config.h [hunk 2]
```diff
@@ -18,52 +18,25 @@
...
-#define MASTER_RIGHT
-
-/* Bootmagic Lite configuration. */
-#define BOOTMAGIC_LITE_ROW 0
-#define BOOTMAGIC_LITE_COLUMN 0
-#define BOOTMAGIC_LITE_ROW_RIGHT 5
-#define BOOTMAGIC_LITE_COLUMN_RIGHT 0
+/* Disable action features. */
+#define NO_ACTION_MACRO    // Disable old-style macro handling.
+#define NO_ACTION_FUNCTION // Disable old-style function handling.
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/4x6/config.h:31]:
```suggestion
```

File: keyboards/bastardkb/skeletyl/config.h [hunk 3]
```diff
@@ -16,87 +17,34 @@
...
 
 #define DIODE_DIRECTION ROW2COL
 
-#define RGB_DI_PIN D2
-#define RGBLED_NUM 36
-#define RGBLED_SPLIT { 18, 18 }
-#define RGBLIGHT_ANIMATIONS
+/* Disable action features. */
+#define NO_ACTION_MACRO    // Disable old-style macro handling.
+#define NO_ACTION_FUNCTION // Disable old-style function handling.
```
Comment (drashna) [inline keyboards/bastardkb/skeletyl/config.h:31]:
```suggestion
```

File: keyboards/bastardkb/tbkmini/config.h [hunk 4]
```diff
@@ -16,90 +17,34 @@
...
 
 #define DIODE_DIRECTION ROW2COL
 
-#define RGB_DI_PIN D2
-#define RGBLED_NUM 42
-#define RGBLED_SPLIT { 21, 21 }
-#define RGBLIGHT_ANIMATIONS
+/* Disable action features. */
+#define NO_ACTION_MACRO    // Disable old-style macro handling.
+#define NO_ACTION_FUNCTION // Disable old-style function handling.
```
Comment (drashna) [inline keyboards/bastardkb/tbkmini/config.h:31]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]
Also, rgb matrix modes removed, without being re-added elsewhere?

File: keyboards/bastardkb/charybdis/3x5/config.h (hunk 1 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x5/config.h:31]: Done.

File: keyboards/bastardkb/charybdis/4x6/config.h (hunk 2 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/4x6/config.h:31]: Done.

File: keyboards/bastardkb/skeletyl/config.h (hunk 3 above)
Comment (0xcharly) [reply keyboards/bastardkb/skeletyl/config.h:31]: Done.

Comment (0xcharly) [reply keyboards/bastardkb/skeletyl/config.h:31]: Done.

**Comment by 0xcharly** [comment]
> Also, rgb matrix modes removed, without being re-added elsewhere?

Correct, they were added back in a follow-up planned PR.  Fixed and added back in this one.

(apologies for the delayed answer.)

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
bump?

File: keyboards/bastardkb/charybdis/3x5/blackpill/rules.mk [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+RGB_MATRIX_DRIVER = WS2812
+
+# Charybdis nano is a split 3x5 keyboard with a maximum of 3 thumb keys (2 on
+# the trackball side).
+SPLIT_KEYBOARD = yes
+LAYOUTS = split_3x5_3 # Support community layout, in particular Manna-Harbour's Miryoku layout
+
+POINTING_DEVICE_ENABLE = yes # Enable trackball
+POINTING_DEVICE_DRIVER = pmw3360
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x5/blackpill/rules.mk:34]:
```suggestion
```

File: keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/config.h [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+            { 0, 0 }
+
+#        ifndef __arm__
+/** Disable unused vial features. */
+
+// In addition to RGB Matrix effects, VialRGB also provides direct LED control
+// with a script running on your computer.  Remove to reenable.
+#            define VIALRGB_NO_DIRECT
+#        endif // __arm__
+#    endif     // VIAL_ENABLE
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/config.h:40]: Vial config needs to be removed.
```suggestion
```

File: keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/vial.json [hunk 7]
```diff
@@ -0,0 +1,76 @@
+{
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/vial.json:1]: vial config needs to be removed

File: keyboards/bastardkb/charybdis/3x5/v2/elitec/rules.mk [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+RGB_MATRIX_DRIVER = WS2812
+
+# Charybdis nano is a split 3x5 keyboard with a maximum of 3 thumb keys (2 on
+# the trackball side).
+SPLIT_KEYBOARD = yes
+LAYOUTS = split_3x5_3 # Support community layout, in particular Manna-Harbour's Miryoku layout
+
+POINTING_DEVICE_ENABLE = yes # Enable trackball
+POINTING_DEVICE_DRIVER = pmw3360
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x5/v2/elitec/rules.mk:33]:
```suggestion
```

File: keyboards/bastardkb/charybdis/3x6/blackpill/rules.mk [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+RGBLIGHT_SUPPORTED = yes    # RGB underglow is supported, but not enabled by default
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix functionality
+RGB_MATRIX_DRIVER = WS2812
+
+SPLIT_KEYBOARD = yes
+LAYOUTS = split_3x6_3
+
+POINTING_DEVICE_ENABLE = yes # Enable trackball
+POINTING_DEVICE_DRIVER = pmw3360
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x6/blackpill/rules.mk:32]:
```suggestion
```

File: keyboards/bastardkb/charybdis/3x6/v1/elitec/rules.mk [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+RGBLIGHT_SUPPORTED = yes    # RGB underglow is supported, but not enabled by default
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix functionality
+RGB_MATRIX_DRIVER = WS2812
+
+SPLIT_KEYBOARD = yes
+LAYOUTS = split_3x6_3
+
+POINTING_DEVICE_ENABLE = yes # Enable trackball
+POINTING_DEVICE_DRIVER = pmw3360
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x6/v1/elitec/rules.mk:31]:
```suggestion
```

File: keyboards/bastardkb/charybdis/3x6/v2/elitec/rules.mk [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+RGBLIGHT_SUPPORTED = yes    # RGB underglow is supported, but not enabled by default
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix functionality
+RGB_MATRIX_DRIVER = WS2812
+
+SPLIT_KEYBOARD = yes
+LAYOUTS = split_3x6_3
+
+POINTING_DEVICE_ENABLE = yes # Enable trackball
+POINTING_DEVICE_DRIVER = pmw3360
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x6/v2/elitec/rules.mk:31]:
```suggestion
```

File: keyboards/bastardkb/charybdis/4x6/blackpill/rules.mk [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix functionality
+RGB_MATRIX_DRIVER = WS2812
+
+# Charybdis is a split 4x6 keyboard with a maximum of 5 thumb keys (3 on the
+# trackball side).
+SPLIT_KEYBOARD = yes
+
+POINTING_DEVICE_ENABLE = yes # Enable trackball
+POINTING_DEVICE_DRIVER = pmw3360
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/4x6/blackpill/rules.mk:33]:
```suggestion
```

File: keyboards/bastardkb/charybdis/4x6/v2/elitec/rules.mk [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix functionality
+RGB_MATRIX_DRIVER = WS2812
+
+# Charybdis is a split 4x6 keyboard with a maximum of 5 thumb keys (3 on the
+# trackball side).
+SPLIT_KEYBOARD = yes
+
+POINTING_DEVICE_ENABLE = yes # Enable trackball
+POINTING_DEVICE_DRIVER = pmw3360
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/4x6/v2/elitec/rules.mk:32]:
```suggestion
```

File: keyboards/bastardkb/dilemma/keymaps/bstiq/config.h [hunk 14]
```diff
@@ -0,0 +1,163 @@
...
+#    ifdef VIAL_ENABLE
+/** Vial-specific configuration. */
+
+#        define VIAL_KEYBOARD_UID \
+            { 0x09, 0x80, 0xDF, 0x80, 0xFB, 0x72, 0xB8, 0x4A }
+#        define VIAL_UNLOCK_COMBO_ROWS \
+            { 0, 4 }
+#        define VIAL_UNLOCK_COMBO_COLS \
+            { 0, 0 }
+#    endif // VIAL_ENABLE
```
Comment (drashna) [inline keyboards/bastardkb/dilemma/keymaps/bstiq/config.h:32]:
```suggestion
```

File: keyboards/bastardkb/scylla/blackpill/rules.mk [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+
+AUDIO_SUPPORTED = no        # Audio is not supported
+RGB_MATRIX_SUPPORTED = yes  # RGB matrix is supported and enabled by default
+RGBLIGHT_SUPPORTED = yes    # RGB underglow is supported, but not enabled by default
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix functionality
+RGB_MATRIX_DRIVER = WS2812  # RGB matrix driver support
+
+SPLIT_KEYBOARD = yes
+
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/scylla/blackpill/rules.mk:29]:
```suggestion
```

File: keyboards/bastardkb/skeletyl/blackpill/rules.mk [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+AUDIO_SUPPORTED = no        # Audio is not supported
+RGB_MATRIX_SUPPORTED = yes  # RGB matrix is supported and enabled by default
+RGBLIGHT_SUPPORTED = yes    # RGB underglow is supported, but not enabled by default
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix functionality
+RGB_MATRIX_DRIVER = WS2812  # RGB matrix driver support
+
+SPLIT_KEYBOARD = yes
+LAYOUTS = split_3x5_3
+
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/skeletyl/blackpill/rules.mk:30]:
```suggestion
```

File: keyboards/bastardkb/tbkmini/blackpill/rules.mk [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+AUDIO_SUPPORTED = no        # Audio is not supported
+RGB_MATRIX_SUPPORTED = yes  # RGB matrix is supported and enabled by default
+RGBLIGHT_SUPPORTED = yes    # RGB underglow is supported, but not enabled by default
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix functionality
+RGB_MATRIX_DRIVER = WS2812  # RGB matrix driver support
+
+SPLIT_KEYBOARD = yes
+LAYOUTS = split_3x6_3
+
+# https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/tbkmini/blackpill/rules.mk:30]:
```suggestion
```

File: keyboards/bastardkb/charybdis/3x5/v1/elitec/rules.mk [hunk 18]
```diff
@@ -36,3 +32,7 @@ POINTING_DEVICE_ENABLE = yes # Enable trackball
 POINTING_DEVICE_DRIVER = pmw3360
 # https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x5/v1/elitec/rules.mk:33]:
```suggestion
```

File: keyboards/bastardkb/charybdis/4x6/v1/elitec/rules.mk [hunk 19]
```diff
@@ -35,3 +31,7 @@ POINTING_DEVICE_ENABLE = yes # Enable trackball
 POINTING_DEVICE_DRIVER = pmw3360
 # https://qmk.fm/changes/2018-11-16-use-a-single-endpoint-for-hid-reports
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/4x6/v1/elitec/rules.mk:32]:
```suggestion
```

File: keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/rules.mk [hunk 20]
```diff
@@ -0,0 +1,12 @@
...
+	TAP_DANCE_ENABLE = no
+	COMBO_ENABLE = no
+	QMK_SETTINGS = no
+else ifeq ($(MCU),STM32F411)
+	EEPROM_DRIVER = vendor
+endif
+
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+VIALRGB_ENABLE = yes
```
Comment (drashna) [inline keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/rules.mk:12]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by 0xcharly** [comment]
A few notes that summaries the various conversations I had with @drashna, @filterpaper, and @karlk90.

## Stability
At this stage, those firmwares are rather well tested and considered stable. They are used daily by multiple members of our community, and are shipped with @bstiq's prebuilds (bastardkb.com).

They have been available through the github.com/bastardkb/bastardkb-qmk fork for close to a year, and have evolved to adopt the latest and greatest from QMK.

## Freshness
To avoid the firmwares becoming stale, our fork is constantly updated against the latest changes that are upstreamed on qmk:master and qmk:develop, and against the latest designs and improvements that @drashna and @karlk90 always kindly notifies us of.

We have been amongst the first few to add support for the STeMCell, and more recently the RP2040.  We have integrated the recent improvements to the pointing device driver, and the latest feature for the Circque trackpad.

## Converters
We've had several discussions with @drashna and @filterpaper w.r.t folder structure and use of converters instead of maintaining several sub folders for each boards.

- While the converters route seems promising, it does not seem to be fully adequate _yet_ for BastardKB boards: we support multiple boards (pro micro, blackpill, elite-c, Splinky, STeMCell) for each keyboards, and while some of these share the same physical footprint (Elite-c/Splinky/STeMCell), these boards do not share the same feature set (mostly because we have to cut down a lot of features on AVR), nor do they always share the same configuration (_eg._ WS2812 driver configuration for the STeMCell). Using a multi folder structure simplifies the maintenance, and the build documentation
- Current converters seem to target the pro micro footprint, which does not include the bottom 5 pins that are available on boards that share the elite-c footprint, which the Bastard Keyboard line use, so converters as they stand today would only partially factorize the configuration, while still requiring per-board setup
- While the current structure and configurations are now well tested and I have a high level of confidence that they work as expected, I would not be able to say the same if we were to make changes now to these keyboard configurations (there are 26 different configurations, and multiple keymaps that would need retesting after such changes)

That being said, I am looking forward to the progress the core team is making with converters, and to being able to take advantage of them for the Bastard Keyboard firmwares. However, I believe this will be best addressed in future PR, to avoid further delaying the upstream of the Bastard Keyboard firmware implementations.

**Comment by 0xcharly** [comment]
Thanks for the review @drashna, let me cleanup the Vial-related stuff.

File: keyboards/bastardkb/charybdis/3x5/blackpill/rules.mk (hunk 5 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x5/blackpill/rules.mk:34]: Done.

File: keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/config.h (hunk 6 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/config.h:40]: Done.

File: keyboards/bastardkb/charybdis/3x6/v1/elitec/rules.mk (hunk 10 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x6/v1/elitec/rules.mk:31]: Done.

File: keyboards/bastardkb/charybdis/4x6/blackpill/rules.mk (hunk 12 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/4x6/blackpill/rules.mk:33]: Done.

File: keyboards/bastardkb/dilemma/keymaps/bstiq/config.h (hunk 14 above)
Comment (0xcharly) [reply keyboards/bastardkb/dilemma/keymaps/bstiq/config.h]: Done.

File: keyboards/bastardkb/skeletyl/blackpill/rules.mk (hunk 16 above)
Comment (0xcharly) [reply keyboards/bastardkb/skeletyl/blackpill/rules.mk]: Done.

File: keyboards/bastardkb/tbkmini/blackpill/rules.mk (hunk 17 above)
Comment (0xcharly) [reply keyboards/bastardkb/tbkmini/blackpill/rules.mk]: Done.

File: keyboards/bastardkb/charybdis/4x6/v1/elitec/rules.mk (hunk 19 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/4x6/v1/elitec/rules.mk]: Done.

File: keyboards/bastardkb/charybdis/3x5/v1/elitec/rules.mk (hunk 18 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x5/v1/elitec/rules.mk]: Done.

File: keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/rules.mk (hunk 20 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/rules.mk]: Done.

File: keyboards/bastardkb/scylla/blackpill/rules.mk (hunk 15 above)
Comment (0xcharly) [reply keyboards/bastardkb/scylla/blackpill/rules.mk]: Done.

File: keyboards/bastardkb/charybdis/4x6/v2/elitec/rules.mk (hunk 13 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/4x6/v2/elitec/rules.mk]: Done.

File: keyboards/bastardkb/charybdis/3x6/v2/elitec/rules.mk (hunk 11 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x6/v2/elitec/rules.mk]: Done.

File: keyboards/bastardkb/charybdis/3x6/blackpill/rules.mk (hunk 9 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x6/blackpill/rules.mk]: Done.

File: keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/vial.json (hunk 7 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x5/keymaps/bstiq/vial.json]: Done.

File: keyboards/bastardkb/charybdis/3x5/v2/elitec/rules.mk (hunk 8 above)
Comment (0xcharly) [reply keyboards/bastardkb/charybdis/3x5/v2/elitec/rules.mk]: Done.

**Review by drashna — APPROVED** [review]

**Review by keyboard-magpie — CHANGES_REQUESTED** [review]
Very impressive work overall, my only request is that the readme level of detail (keyboards/bastardkb/charybdis/3x6/keymaps/via/readme.md) for charybdis 3x6 is replicated  for the 3x5 and 4x6 via keymaps as they are complex keymaps with key features that should be highlighted to the end user, it should be as simple as duplicating the readme for the 3x6 via keymap.

**Review by keyboard-magpie — APPROVED** [review]

**Comment by bstiq** [comment]
Excellent work @0xcharly , thanks !
Looks good to me.

**Pull request merged**